sources/packages/api/src-sql/Restrictions/Clean-restriction-add-action-and-notification-type-remove-allowed-count-and-seed.sql

```diff
@@ -0,0 +1,1326 @@
+-- Clean restrictions table.
```

Comment:
file name seems to be way too lengthy(86 characters) if possible it can be brief.

Reply:
IMO, unless there is a technical constraint I would not mind to leave as it is.